invenio_communities/assets/semantic-ui/js/invenio_communities/requests/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import _camelCase from "lodash/camelCase";
+import { overrideStore } from "react-overridable";
+import { SearchApp } from "@js/invenio_search_ui/components";
+import { defaultComponents as RequestsDefaultComponents } from "./requests";
+
+const domContainer = document.getElementById("app");
+const community = JSON.parse(domContainer.dataset.community);
+
+const getConfigFromDataAttribute = (element, attr) => {
+  const dataValue = domContainer.dataset[attr];
+
+  return JSON.parse(dataValue);
+};
+
+class CommunityRequests extends Component {
+  render() {
+    const { appId, ...config } = getConfigFromDataAttribute(
+      domContainer,
+      _camelCase("invenio-search-community-requests-config")
+    );
+    config["initialQueryState"]["filters"].push([
+      "receiver",
+      "community:" + community.uuid,
+    ]);
+    for (const [componentId, component] of Object.entries(
+      RequestsDefaultComponents
+    )) {
+      overrideStore.add(componentId, component);
```

Comment:
why is this needed?

Reply:
As it was done for the dashboard, this piece of code overrides the content that will be displayed by the SearchApp component. In this case, the overrides of the components are defined here: https://github.com/inveniosoftware/invenio-communities/pull/455/files#diff-6662c287617406713a1a6b49085ecdf7fccf1e16f2b54c2635b574ff3be8684fR527-R540.